enginetest/enginetests.go

```diff
@@ -540,6 +540,549 @@ func TestDeleteErrors(t *testing.T, harness Harness) {
 	}
 }
 
+func TestTruncate(t *testing.T, harness Harness) {
+	e := NewEngine(t, harness)
+	ctx := NewContext(harness)
+
+	t.Run("Standard TRUNCATE", func(t *testing.T) {
+		_, iter, err := e.Query(ctx, "CREATE TABLE t1 (pk BIGINT PRIMARY KEY, v1 BIGINT, INDEX(v1))")
```

Comment:
Every place in this file where you're just asserting that no error occurs from a query without checking the result, use RunQuery instead. Saves you a bunch of boilerplate lines (and reduces copypasta errors)